File: 0344-reverse-string/python/Solution.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        n = len(s)
        i = 0
        j = n - 1
        while i <= j:
            tmp = s[i]
            s[i] = s[j]
            s[j] = tmp
            i += 1
            j -= 1

File: 0344-reverse-string/javascript/Solution.js
/**
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
var reverseString = function(s) {
    const n = s.length
    let i = 0
    let j = n - 1
    while (i <= j) {
        [s[i], s[j]] = [s[j], s[i]]
        i++
        j--
    }
};